.github/workflows/CI.yml

```diff
@@ -194,7 +194,7 @@ jobs:
                    if [[ $TEST_SUITES == *testbuildsys* ]] ; then
                        sudo apt-get remove libgmp-dev libreadline-dev zlib1g-dev
                    else
-                       packages+=(libgmp-dev libreadline-dev zlib1g-dev expect)
+                       packages+=(libgmp-dev libreadline-dev zlib1g-dev pkg-config expect)
```

Comment:
My understanding is that there are problems in one 32bit CI job; the error says:
```
The following packages have unmet dependencies:
 pkg-config:i386 : Depends: libglib2.0-0:i386 (>= 2.16.0) but it is not going to be installed
```
First off, this kind of annoyance is one of the reasons why @ChrisJefferson and me would like to just drop 32bit support. Yes, this can be fixed and worked around, but it always costs us.

In this particular case, the "problem" is kinda the code coming a bit later, in lines 222ff: If building in 32bit mode, it asks for *all* packages to install the 32bit counterparts.

But that's not really necessary, of course: only the various `lib*` packages (or anything installing a shared library we want to link against, really) needs this treatment. But `pkg-config` (and singular, pari-gp, texlive, ...) surely do not. 

So one could split the `packages` list into two, one to which the 32bit transformation is applied, and one that is not.

Or for this particular case, a much simpler solution: just don't add `pkg-config` here. Instead, insert `sudo apt-get install pkg-config` after line 230 or so

Reply:
done